draftlogs/7115_change.md

```diff
@@ -0,0 +1,2 @@
+ - Update turf.js to v7 [[#7115](https://github.com/plotly/plotly.js/pull/7115)]
```

Comment:
Let's rename the filename and log as a bug fix.

Reply:
There is bugs in the multi polygon test as well, that I'm investigating

Reply:
I have updated the draftlog